### DISCUSSION TITLE
[Improvement]: Tabbed Navigation for Concurrent Requests in Insomnia

### DISCUSSION THREAD

**Discussion by gustavomoda** [Kong/insomnia#7326] [Ideas]
Hey Insomnia Team,

Hope you're all doing well! I've been using Insomnia for a while now and absolutely love it. However, I have a cool idea that might make our lives a bit easier. How about adding tabbed navigation for handling multiple requests at once? Right now, flipping through the list to get back to previous requests can be a bit of a drag. Implementing tabs would allow us to switch between them seamlessly, just like we do in web browsers. It would definitely boost our workflow and make the UI even more intuitive.

Thanks for considering this tweak. I’m here if you want to chat more about it!

Cheers,

**Comment by subnetmarco** [comment]
Yes we really want to build this, and we will start soon. Ideally in the next few months we will have it. Closing this discussion in favor of https://github.com/Kong/insomnia/discussions/6108
